Makefile

```diff
@@ -4,51 +4,54 @@ IDIR = include
 ODIR = obj
 
 INSTALL = install
-PREFIX = /usr/bin
+PREFIX = /usr
 
-MANDIR = /usr/share/man/man1
+BINDIR = $(PREFIX)/bin
+MANDIR = $(PREFIX)/share/man/man1
 
 CC = gcc
-CFLAGS += -I$(IDIR)
+LD = $(CC)
+
+CPPFLAGS += -D'__VERSION="$(shell git describe --all --long --always)"' "-I$(IDIR)"
+
 CFLAGS += -std=gnu99
 CFLAGS += -Wall -Wundef -Wshadow -Wformat-security
-LIBS = $(shell pkg-config --libs x11 xi xfixes)
+
+LDFLAGS += $(shell pkg-config --libs x11 xi xfixes)
 # libev has no pkg-config support
-LIBS += -lev
+LDFLAGS += -lev
 
 INCS = $(wildcard $(IDIR)/*.h)
 SRCS = $(wildcard $(SDIR)/*.c)
 OBJS = $(patsubst %,$(ODIR)/%,$(notdir $(SRCS:.c=.o)))
 
 MANS = man/unclutter-xfixes.1
 
-.NOTPARALLEL:
-
 .PHONY: all
-all: clean $(TARGET) mans
+all: $(TARGET)
```

Comment:
Hm, why remove the dependency to `clean` here?

Reply:
As far as I know the common convention for the phony target `all` is to build everything that needs to be built and is not a synonym for "rebuild everything", which the dependency to clean would imply.

For reference I had a quick look at the [GNU Coding Standards](https://www.gnu.org/software/make/manual/html_node/Standard-Targets.html#Standard-Targets), but, unfortunately, this description isn't explicit about it.

Consulting the [provided example](https://www.gnu.org/software/make/manual/html_node/Complex-Makefile.html#Complex-Makefile) reveals the same assumption I made, though.

Reply:
My issue with it is that changing this might cause build scripts for packages etc. to stop working as expected now. Given that this doesn't really relate to cross-compiling anyway, I think I'd prefer if we could remove this particular change. Can we do that?